BlogTemplate.Tests/Models/BlogDataStoreTests.cs

```diff
@@ -42,8 +42,8 @@ public void SaveComment_SaveSimpleComment()
                 Slug = "Test-slug",
                 Title = "Test title",
                 Body = "Test body",
-                PubDate = DateTime.Now,
-                LastModified = DateTime.Now,
+                PubDate = DateTimeOffset.UtcNow,

```

Comment:
With DateTimeOffset, it doesn't matter if you use .Now or .UtcNow, because the offset is stored in the value.  This means that `DateTimeOffset.Now == DateTimeOffset.UtcNow` (as opposed to `DateTime.Now != DateTime.UtcNow`).

Of course, servers will likely be running in UTC time anyways, so it doesn't matter that much, and even less so since these are test values.